### PR TITLE
Add actions for Google Drive integration

### PR DESCRIPTION
**Pull request by farmanrl**
tests coming soon

### CODE REVIEW

**Comment by farmanrl** [comment]
I added a test for the reducer, but neglected the actionCreators since we'll be revamping those entirely when we add Google Drive functionality

File: src/redux/actionCreators.js [hunk 1]
```diff
@@ -69,7 +69,60 @@ export function changeSource(source) {
...
+  };
+}
+
+export function connectGoogleDrive() {
+  return dispatch => {
+    dispatch({type: actType.START_CONNECT_DRIVE});
+    var promise = new Promise((resolve, reject) => {
+      var googleDrive = "Google Drive";
+      window.setTimeout(() => resolve(googleDrive), 1000);
+      reject(new Error("Google Drive not connected"));
```
Comment (pcardune) [inline src/redux/actionCreators.js]: It seems kind of strange that these all "fail" and then "succeed" a second later.

**Comment by pcardune** [comment]
Fix those tests!

**Comment by pcardune** [comment]
Looks good! just rebase and I'll merge.

**Pull request merged**